scripts/apps/analytics/directives/SavedActivityReports.js

```diff
@@ -0,0 +1,119 @@
+SavedActivityReports.$inject = [
+    '$rootScope', 'api', 'session', 'modal', 'notify', 'gettext', 'asset',
+    '$location', 'desks', 'privileges', 'config', 'savedActivityReports'
+];
+
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.analytics
+ * @name sdSavedActivityReports
+ * @requires $rootScope, api, session, modal, notify, gettext, asset, $location, desks,
```

Comment:
I don't think this works. It'll have to be a separate line for each one, like in the other modules.
```js
@requires $rootScope
@requires api
// .. etc
```